autoplex/fitting/common/utils.py

```diff
@@ -249,7 +248,7 @@ def gap_fitting(
 def jace_fitting(
     db_dir: str | Path,
     path_to_default_hyperparameters: Path | str = MLIP_RSS_DEFAULTS_FILE_PATH,
```

Comment:
My suggestion here would be to remove the default file paths here altogether. I think it is confusing to have both a RSS and a phonon default here. Maybe, we can add this as an issue. I don't think this is an urgent request.

Reply:
This is what I'm thinking: as we discussed earlier, we can't isolate the GAP settings from other models. I believe we've already identified the relevant issues (https://github.com/autoatml/autoplex/issues/91). However, for now, it seems everyone is aligned. We can proceed with adjustments once all tests for the rss and phonon workflow have successfully passed.